src/Components/Loader/Loader.scss

```diff
@@ -0,0 +1,75 @@
+html, body {
+    height: 100%;
+  }
+
+#cube-loader {
```

Comment:
Copy pasted?!
You could do it better, but OK))

Reply:
Do you want me to do it by myself?)